<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>personalprofilescreen</class>
 <widget class="QWidget" name="personalprofilescreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
</string>
   </property>
   <property name="text">
    <string>‚Üê</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="bolusButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 32pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>0 hrs</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Bolus Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Press to Set Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Timed Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 32pt &quot;DejaVu Sans&quot;;
text-align: left;

</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>DURATION</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>CARBS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>190</y>
       <width>41</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 32pt &quot;DejaVu Sans&quot;;
text-align: left;
background-color: rgb(153, 193, 241);

</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>185</y>
       <width>31</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_6">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>41</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 32pt &quot;DejaVu Sans&quot;;
text-align: left;
background-color: rgb(153, 193, 241);

</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>436</x>
       <y>55</y>
       <width>29</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 40pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>_</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Profile: 0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 13pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Select Profile</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="bolusButton_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Basal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Correction Factor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Carb Ratio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Target BG</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="basalRate">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Enter Number Here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="correctionFactor">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Enter Number Here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="carbRatio">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Enter Number Here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="targetBG">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Enter Number Here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeHourValue">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="timeMinuteValue">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>12</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 18pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QLabel" name="timeLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>13</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>38</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>38</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>Mins</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_4">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>33</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>(24h Format)</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>u/hr</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>mmol/L</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>g</string>
     </property>
    </widget>
    <widget class="QLabel" name="setActiveLabel_8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>240</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="text">
      <string>mmol/L</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="bolusButton_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Insulin Duration</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="insulinDuration">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QPushButton" name="bolusButton_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;DejaVu Sans&quot;;

</string>
     </property>
     <property name="text">
      <string>Carbohydrates</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="carboCheckBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QSpinBox" name="insulinDuration_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Per Profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addTimeslotButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 25pt &quot;DejaVu Sans&quot;;
background-color: rgb(119, 118, 123);
</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="homeButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(119, 118, 123);
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:10pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; font-weight:400;&quot;&gt;Enter Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(143, 240, 164);
</string>
   </property>
   <property name="text">
    <string>‚úì</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(143, 240, 164);
</string>
   </property>
   <property name="text">
    <string>‚úì</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(246, 97, 81);
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="deleteLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(143, 240, 164);
</string>
   </property>
   <property name="text">
    <string>‚úì</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>200</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(246, 97, 81);
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="deleteLabel_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setActiveButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 35pt &quot;DejaVu Sans&quot;;
background-color: rgb(143, 240, 164);
</string>
   </property>
   <property name="text">
    <string>‚úì</string>
   </property>
  </widget>
  <widget class="QLabel" name="setActiveLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Set As Active</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
